libraries/rush-sdk/src/index.ts

```diff
@@ -31,6 +31,16 @@ declare const global: NodeJS.Global &
     ___rush___rushLibModuleFromInstallAndRunRush?: RushLibModuleType;
   };
 
+// eslint-disable-next-line @typescript-eslint/naming-convention,@typescript-eslint/no-explicit-any
+declare const __non_webpack_require__: ((moduleName: string) => any) | undefined;
```

Comment:
Can this go in a library?  At the very least it needs code comments explaining (1) what it does, (2) why we need it, (3) how it works.